docs/guides-buidler-migration.md

```diff
@@ -0,0 +1,193 @@
+
+For this tutorial, we will use the original [react boilerplate for aragonCLI](https://github.com/aragon/aragon-react-boilerplate/tree/react-with-cli) source code as our starting point. This guide assumes that you have a general understanding of the Aragon stack. 
+
+## 1. Install dependencies
+
+The first step is to install the following NPM development dependencies:
+
+  - **"@aragon/buidler-aragon"**: "^0.1.0"
+  - **"@aragon/contract-test-helpers"**: "0.0.1"
+  - **"@nomiclabs/buidler"**: "^1.0.2"
+  - **"@nomiclabs/buidler-truffle5"**: "^1.1.2"
+  - **"@nomiclabs/buidler-web3"**: "^1.1.2"
+  - **"bignumber.js"**: "^9.0.0"
+  - **"web3"**: "^1.2.6"
+
+Most of these dependencies should be new to your project and are related to Nomic Labs' [Buidler](https://buidler.dev), a task runner for Ethereum smart contract developers. However, one dependency that you may need to upgrade is `web3`, since Aragon's plugin requires `v1.2.6` or higher. This may introduce some breaking changes, mostly in tests. See the [Migrate tests](#5-migrate-tests) section for more information.
+
+## 2. Add/replace npm scripts
+
+Since the app won't use `aragonCLI` anymore, some NPM scripts need to be changed.
+
+In `package.json`:
+
+  - "postinstall": "npm run build-app"
+  - "build-app": "cd app && npm install && cd .."
+  - "compile": "buidler compile --force"
+  - "start": "buidler start"
+  - "test": "buidler test --network buidlerevm"
+
+Also, make sure that you have the following two scripts in `app/package.json`:
+
+  - **serve**: Launches the web server (Previously called `devserver` script in most projects)
+  - **watch**: Watches for file changes (Previously called `watch:script` script in most projects)
+
+
+## 3. Update `.gitignore` file
+
+Add the following two folders to your `.gitignore` file:
+
+  - artifacts
+  - cache
+
+## 4. Create the Buidler config file
+
+Add the following `buidler.config.js` file to the root of your project: 
+
+```js
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin('@aragon/buidler-aragon')
+
+module.exports = {
+  defaultNetwork: 'localhost',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545'
+    },
+  },
+  solc: {
+    version: '0.4.24',
+    optimizer: {
+      enabled: true,
+      runs: 10000
+    }
+  },
+  aragon: {
+    appServePort: 1234,
+    clientServePort: 3000,
+    appSrcPath: 'app/',
+    appBuildOutputPath: 'dist/',
+    hooks: require('./scripts/buidler-hooks')
+  }
+}
+```
+
+You can find more information about the Buidler configuration file [here](https://buidler.dev/config). As for the `aragon` section, it contains 5 main options:
```

Comment:
We should remember to link this section to the documentation we'll be writing for the buidler plugin :).